### DISCUSSION TITLE
Migrating to new server. After migration login page will not load on new server

### DISCUSSION THREAD

**Discussion by damion-dish** [snipe/snipe-it#13210] [Q&A]
I have migrated a SnipeIT database from one server to another. I have edited and configured the the copied config files for the new server but now am unable to get the login page to load. I am getting a 404 but have the page enabled with the correct user and I only getting a 404. I haven't been able to get the login page to load at all on the new server.
![Screenshot 2023-06-26 at 3 42 55 PM](https://github.com/snipe/snipe-it/assets/109315839/44a7bb06-0554-4347-856c-9e1e48d92a9b)
x
![Screenshot 2023-06-26 at 3 50 42 PM](https://github.com/snipe/snipe-it/assets/109315839/103b7f10-d558-4c03-8312-e9551cd5bd90)

**Comment by snipe** [comment]
Sounds like probably your document root is not set correctly in your web server.
